packages/property-view/src/browser/resource-property-view/resource-property-view-widget-provider.ts

```diff
@@ -20,6 +20,11 @@ import { inject, injectable } from '@theia/core/shared/inversify';
 import { DefaultPropertyViewWidgetProvider } from '../property-view-widget-provider';
 import { ResourcePropertyViewTreeWidget } from './resource-property-view-tree-widget';
 
+/**
+ * The provider extends the {@link DefaultPropertyViewWidgetProvider}
```

Comment:
I would omit the first line as it does not offer additional information that the class does not already provide with:

```ts
export class ResourcePropertyViewWidgetProvider extends DefaultPropertyViewWidgetProvider {
```